auth-web/src/views/auth/MaintainBusinessView.vue

```diff
@@ -114,5 +120,15 @@ export default class MaintainBusinessView extends Vue {
         font-weight: bold;
       }
     }
+
+    .create-account-link {
+      font-size: 1rem;
+      font-weight: bold;
+      color: $BCgoveBueText1!important;
```

Comment:
Lint is OK without a space between the value and "!important"?
Would you mind adding a space anyway? Thanks!

Reply:
hmm, yes it seems ok. Will definitely add a space tho, 100%

Reply:
Updated in next commit

Reply:
Resolved.